### PR TITLE
DO-194: Fixed issues with family details edit page.

### PR DESCRIPTION
**Pull request by dhruvlahoti**

### CODE REVIEW

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx [hunk 1]
```diff
@@ -34,10 +33,11 @@ const ArrayTemplateForm = (props) => {
   const heading = `${element.children.props.formData.name}, ${element.children.props.formData.relationshipType}`;
 
   const renderHeading = () => {
+    // FIXME: Should use Paragraph component of Typography
```
Comment (ayushmalik2110) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx:36]: When ??

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx [hunk 2]
```diff
@@ -34,10 +33,11 @@ const ArrayTemplateForm = (props) => {
...
     if (element.children.props.formData.name) {
-      return <Paragraph>{heading}<span><Icon icon={isEditClicked ? faMinus : faPlus} /></span></Paragraph>;
+      return <p>{heading}<span><Icon icon={isEditClicked ? faMinus : faPlus} /></span></p>;
```
Comment (ayushmalik2110) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx:38]: Why you removed <Paragraph />

Comment (ayushmalik2110) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx:38]: If typography component have some issue with children then resolved it.

Comment (dhruvlahoti) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx:38]: There is some styling Problem. If we are using Paragraph component the Paragraph is not visible on screen, as it is a potential Blocker I reverted it to <p>

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx (hunk 1 above)
Comment (dhruvlahoti) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx:36]: Will do it after beta release

**Pull request merged**